pkg/cmd/cmd.go

```diff
@@ -302,20 +302,28 @@ func fullPath(command *cobra.Command) string {
 }
 
 func setLoggingLevel(cmd *cobra.Command, args []string) {
-	verbose, err := strconv.ParseBool(cmd.Flag(opts.OptionVerbose).Value.String())
-	if err != nil {
-		log.Logger().Errorf("Unable to determine log level")
-	}
-
-	if verbose {
+	level := os.Getenv("JX_LOG_LEVEL")
+	if level != "" {
 		err := log.SetLevel("debug")
```

Comment:
> That's not right. This would hard code the log level to debug if JX_LOG_LEVEL is set.

Ouch. Thanks for catching it.

> Could you maybe add a test which verifies that the code works as expected?

Ok, I'll do it.